mmv1/products/vertexai/api.yaml

```diff
@@ -277,6 +277,59 @@ objects:
+                description: |
+                  Customized export features time window for snapshot analysis. Unit is one day. The default value is 21 days. Minimum value is 1 day. Maximum value is 4000 days.
+                default_value: 21
+          - !ruby/object:Api::Type::NestedObject
+            name: 'importFeaturesAnalysis'
+            description: |
+              The config for ImportFeatures Analysis Based Feature Monitoring.
+            properties:
+              - !ruby/object:Api::Type::String
+                name: state
+                description: |
+                  Whether to enable / disable / inherite default hebavior for import features analysis. The value must be one of the values below:
+                  * STATE_UNSPECIFIED: Should not be used.
+                  * DEFAULT: The default behavior of whether to enable the monitoring. EntityType-level config: disabled.
+                  * ENABLED: Explicitly enables import features analysis. EntityType-level config: by default enables import features analysis for all Features under it.
+                  * DISABLED: Explicitly disables import features analysis. EntityType-level config: by default disables import features analysis for all Features under it.
+              - !ruby/object:Api::Type::String
+                name: 'anomalyDetectionBaseline'
+                description: |
+                  Defines the baseline to do anomaly detection for feature values imported by each [entityTypes.importFeatureValues][] operation. The value must be one of the values below:
+                  * BASELINE_UNSPECIFIED: Should not be used.
+                  * LATEST_STATS: Choose the later one statistics generated by either most recent snapshot analysis or previous import features analysis. If non of them exists, skip anomaly detection and only generate a statistics.
+                  * MOST_RECENT_SNAPSHOT_STATS: Use the statistics generated by the most recent snapshot analysis if exists.
+                  * PREVIOUS_IMPORT_FEATURES_STATS: Use the statistics generated by the previous import features analysis if exists.
+          - !ruby/object:Api::Type::NestedObject
+            name: 'numericalThresholdConfig'
+            description: |
+              Threshold for numerical features of anomaly detection. This is shared by all objectives of Featurestore Monitoring for numerical features (i.e. Features with type (Feature.ValueType) DOUBLE or INT64).
+            properties:
+              - !ruby/object:Api::Type::Double
+                name: 'value'
+                description: |
+                  Specify a threshold value that can trigger the alert. For numerical feature, the distribution distance is calculated by Jensen–Shannon divergence. Each feature must have a non-zero threshold if they need to be monitored. Otherwise no alert will be triggered for that feature. The default value is 0.3.
+                default_value: 0.3
```

Comment:
I think we want to make this `required` rather than having a default, as we don't want `numerical_threshold_config` to be an empty block. Same with `categorical_threshold_config` below. What are your thoughts?

Reply:
If I remember correctly, the API returned 0.3 when not specifying the fields. That said, I agree. we don't want them to be empty fields. I'll make them required. Thanks!